synchro/synchrotest.py

```diff
@@ -356,8 +356,12 @@ def want_method(method, classname):
 
     # Run the tests from the pymongo target dir with our custom plugin.
     os.chdir(sys.argv[1])
+    if "-m" in sys.argv[2:]:
+        markers = []
+    else:
+        markers = ["-m", "default"]
     code = pytest.main(
-        sys.argv[2:] + ["-m", "default"] + ["-p", "no:warnings"], plugins=[SynchroPytestPlugin()]
+        sys.argv[2:] + markers + ["-p", "no:warnings"], plugins=[SynchroPytestPlugin()]
```

Comment:
This skips the pymongo async tests right?

Reply:
You can't supply two `-m` flags, only the last one is used.  So we allow the one passed in on `sys.argv` to override.

Reply:
Looks good to me.